app/models/SuperModel.coffee

```diff
@@ -132,8 +132,11 @@ module.exports = class SuperModel extends Backbone.Model
     # can't use instanceof. SuperModel gets passed between windows, and one window
     # will have different class objects than another window.
     # So compare className instead.
-    return (m for key, m of @models when m.constructor.className is ModelClass.className) if ModelClass
-    return _.values @models
+    if not ModelClass
+      return _.values @models
+    # Allow checking by string name to reduce module dependencies
+    className = if _.isString(ModelClass) then ModelClass else ModelClass.className
+    return (m for key, m of @models when m.constructor.className is className)
```

Comment:
Needs real code review. Refactored slightly, updated to allow for passing a string instead of an object (to reduce dependencies)

Reply:
There are other places that could have `ThangType` switched to `"ThangType"`, but they all look like they would use the game libraries anyway. Is this true? If so, this looks good to me.